### PR TITLE
Update schema files

### PR DESCRIPTION
**Pull request by tejaswinichile**
Update schema files

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -281,6 +273,28 @@
     t.index ["phone_number"], name: "index_channel_whatsapp_on_phone_number", unique: true
   end
 
+  create_table "companies", force: :cascade do |t|
```
Comment (pranavrajs) [inline db/schema.rb:276]: We don't have companies table yet, we might need another PR to clean this up  😆

Comment (tejaswinichile) [reply db/schema.rb:276]: @pranavrajs Ohh is it my changes? Let me check this. :D

**Comment by tejaswinichile** [comment]
@sojan-official This https://github.com/chatwoot/chatwoot/pull/3438#issuecomment-975396972 is a good approach, I am gonna do it `actions` table.
Any idea why Am I getting companies table changes?

**Comment by sojan-official** [comment]
@tejaswinichile  you might have the companies table in your local database from an older migration. Whenever you run `rake db:migrate` rails will create a new schema file with your local database copy.  Two options to fix this is

- if you have data in your local you don't want to lose, delete the table using raw SQL
- or  checkout develop and do a rake db:reset

**Comment by tejaswinichile** [comment]
@sojan-official Oh yes! now I remember that I was working on custom attributes to save in companies table like 2-3 months back 😁  Yes, we don't need this change in development yet.

File: db/schema.rb [hunk 2]
```diff
@@ -326,7 +318,7 @@
     t.datetime "agent_last_seen_at"
     t.jsonb "additional_attributes", default: {}
     t.bigint "contact_inbox_id"
-    t.uuid "uuid", default: -> { "public.gen_random_uuid()" }, null: false
+    t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
```
Comment (sojan-official) [inline db/schema.rb:321]: this change doesn't seem intended.

File: db/schema.rb [hunk 3]
```diff
@@ -326,7 +318,6 @@
     t.datetime "agent_last_seen_at"
     t.jsonb "additional_attributes", default: {}
     t.bigint "contact_inbox_id"
-    t.uuid "uuid", default: -> { "public.gen_random_uuid()" }, null: false
```
Comment (sojan-official) [inline db/schema.rb:329]: @tejaswinichile 😅 this is needed though

Comment (tejaswinichile) [reply db/schema.rb:329]: I just did reset :D

**Pull request merged**